templates/error.html

```diff
@@ -0,0 +1,139 @@
+{% extends 'base.htm' %}
+
+{% block content %}
```

Comment:
Indentation is inconsistent; can you format the file?  PyCharm might be able to autoindent it for you.  Our standard is 4 spaces per indentation.

Reply:
Ah, there were just some tabs thrown in there.